lib/bootstrap-bookingsync-sass.rb

```diff
@@ -10,9 +11,16 @@ def load!
           register_rails_engine
         elsif sprockets?
           register_sprockets
+        elsif defined?(::Sass) && ::Sass.respond_to?(:load_paths)
+          # The deprecated `sass` gem:
```

Comment:
Seems like I'm criticizing my previous changes, but if it's deprecated, and we already have sassc dependency, it might be worth to drop it and rely on sassc completely?  https://github.com/sass/sassc-ruby/blob/e7da015b3aa535b4045ca28be8f41a8077321007/lib/sassc.rb#L21

Reply:
this is used on `bootstrap-sass` itself and can help when using outside of rails, with nanoc for the doc for example.

Reply:
> this is used on bootstrap-sass

bootstrap-sass changed dependency to sassc on version 3.4.0, so might may more sense to just lock it on that version. 
But having that requirement by nanoc is indeed complicating it